keras/layers/containers.py

```diff
@@ -488,6 +488,7 @@ def add_shared_node(self, layer, name, inputs=[], merge_mode=None,
                 sh = SiameseHead(i)
                 sh.previous = s
                 sh_name = outputs[i]
+                sh.name =sh_name
```

Comment:
Correct spaces, please. Then squash your commits so you only have one commit for this PR.